Kudu.Services/FetchHelpers/OneDriveHelper.cs

```diff
@@ -62,7 +62,7 @@ internal ILogger Logger
             ChangeSet changeSet = null;
             string cursor = _settings.GetValue(CursorKey);
             ChangesResult changes = null;
-            using (_tracer.Step("Getting delta changes with cursor: {0} ...", cursor))
+            using (_tracer.Step("Getting delta changes with cursor: {0}...", cursor.Truncate(5)))
```

Comment:
Fixed in https://github.com/projectkudu/kudu/pull/2040/commits/e0e95d54acae05896e456e1154660deca178613e
![image](https://cloud.githubusercontent.com/assets/6472374/15984606/5c14b6f0-2fda-11e6-822f-3d52b148a20f.png)